easybuild/easyconfigs/q/QEMU/QEMU-2.5.0-foss-2016a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+name = 'QEMU'
+version = '2.5.0'
+
+homepage = 'http://www.qemu.org/'
+description = "QEMU: CPU emulator using dynamic translation"
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://wiki.qemu-project.org/download/']
+configopts = "--disable-rdma --enable-kvm --enable-docs --disable-gtk --enable-linux-aio --enable-virtfs"
+
+osdependencies = [
+    ('libattr1-dev', 'libattr-devel'),
+    ('libcap-dev', 'libcap-devel'),
+    ('libaio-dev', 'libaio-devel'),
+    ('texinfo'),
+    ('libglib2.0-dev', 'glib2-devel')
```

Comment:
several of these can be resolved through EasyBuild instead, which would significantly help imho...